proxy-node-shared/src/test/java/uk/gov/ida/notification/shared/proxy/VSPProxyGenerateRequestTest.java

```diff
@@ -106,12 +106,12 @@ public void shouldThrowVerifyServiceProviderResponseExceptionOnServerError() {
             fail("Expected exception not thrown");
         } catch (VspGenerateAuthnRequestResponseException e) {
             assertThat(e.getCause().getMessage())
-                .startsWith(
-                    String.format(
-                        "Exception of type [REMOTE_SERVER_ERROR] whilst contacting uri: %s/generate-request",
-                        testVspServerErrorClientRule.baseUri().toString()
-                    )
-                );
+                    .startsWith(
```

Comment:
When testing for exceptions thrown, AssertJ has a mechanism to capture and check expected properties - these should be used instead of try/catch. Examples:
```
assertThatThrownBy(() -> { throw new Exception("boom!"); }).isInstanceOf(Exception.class)
                                                             .hasMessageContaining("boom");
```
```
assertThatExceptionOfType(IOException.class).isThrownBy(() -> ...
```
```
   // when
   Throwable thrown = catchThrowable(() -> { throw new Exception("boom!"); });

   // then
   assertThat(thrown).isInstanceOf(Exception.class)
```

Guidance here: http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion

Can we please re-write these tests to use the testing framework methods?

Reply:
Can disregard the above for this PR as these tests were written previously in a separate PR

Reply:
Will disregard.